pk-test/docker.lua

```diff
@@ -0,0 +1,191 @@
+--------------------------------------------------------------------------------
+-- docker.lua
+-- This file is a part of pk-test library
+-- Copyright (c) pk-test authors (see file COPYRIGHT for the license)
+-- See file `COPYRIGHT` for the license
+--------------------------------------------------------------------------------
+
+require 'lua-aplicado.module' -- for import()
+require 'lfs' -- for lfs.chdir(), lfs.currentdir()
+
+-----------------------------------------------------------------------
+
+local shell_exec
+      = import 'lua-aplicado/shell.lua'
+      {
+        'shell_exec'
+      }
+
+local arguments
+      = import 'lua-nucleo/args.lua'
+      {
+        'arguments'
+      }
+
+-----------------------------------------------------------------------
+
+local log, dbg, spam, log_error
+      = import 'lua-aplicado/log.lua' { 'make_loggers' } (
+          'test/docker', 'TDKR'
+        )
+
+-----------------------------------------------------------------------
+
+--[[
+what function does:
+  starts docker container, calls handler, stops container after
+  handler's end.
+how does function handle errors:
+  function currently crashes with calling error() on
+  any error.
+function returns true on success. ]]
+local function do_with_docker (cfg_dir, handler)
+  arguments(
+      'string', cfg_dir,
+      'function', handler
+    )
```

Comment:
archaic indentation, please use modern one

Reply:
please, point on modern rules of indentation for arguments()

(I used these:

https://github.com/logiceditor-com/coding-guidelines/blob/master/lua-guidelines.md#%D1%88%D1%82%D0%B0%D1%82%D0%BD%D0%B0%D1%8F-%D0%BF%D1%80%D0%BE%D0%B2%D0%B5%D1%80%D0%BA%D0%B0-%D0%B0%D1%80%D0%B3%D1%83%D0%BC%D0%B5%D0%BD%D1%82%D0%BE%D0%B2
)

Reply:
```
foo(
  bar,
  baz
)
```

not

```
foo(
    bar,
    baz
  )
```